lib/blue_bird/writer/blueprint.ex

```diff
@@ -234,6 +236,38 @@ defmodule BlueBird.Writer.Blueprint do
   defp print_route_description(nil), do: ""
   defp print_route_description(description), do: "#{description}\n"
 
+  ## Attributes
+
+  @spec print_attributes(map) :: String.t
+  def print_attributes(attributes) when attributes == %{}, do: ""
+  def print_attributes(attributes) do
+    "+ Attributes (object)\n\n"
+    <> walk_through_attributes(attributes)
+  end
+
+  defp print_attribute({key, value}) when is_map(value) do
+    print_attribute_name_and_type(key, value)
+    <> walk_through_attributes(value, 4)
+  end
+
+  defp print_attribute({key, value}) when is_list(value) do
+    if is_map(List.first(value)) do
+      print_attribute_name_and_type(key, value)
+      <> ("+ (object)\n" |> indent(4))
+      <> walk_through_attributes(List.first(value))
+    else
+      print_attribute_name_and_type(key, value)
+    end
+  end
+
+  defp print_attribute({key, value}), do: print_attribute_name_and_type(key, value)
```

Comment:
Too long lines.

Reply:
Can you check the credo configuration? It doesn't emit a warning here.

Reply:
Also, there's a credo warning for `generator.ex` line 176, which you didn't change, so I wonder why it went through Travis without notice.

Reply:
I do not have any credo warnings at all.... I'm always using credo with `--strict`...